src/FileLoader/structs/BMPFile.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Shared.Structs;
+
+namespace FileLoader.structs
+{
+    public struct BMPFile
+    {
+        public int Width { get; set; }
```

Comment:
We can save some memory by putting `UInt16` there and in height, but it is a not a must have. In this case `Size` should be `UInt32` also.